src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationDashboard.tsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+import styled from 'react-emotion';
+
+import AsyncView from 'app/views/asyncView';
+import SettingsPageHeader from 'app/views/settings/components/settingsPageHeader';
+import StackedBarChart from 'app/components/stackedBarChart';
+import {Panel, PanelBody, PanelHeader, PanelItem} from 'app/components/panels';
+import DateTime from 'app/components/dateTime';
+import LoadingError from 'app/components/loadingError';
+import EmptyMessage from 'app/views/settings/components/emptyMessage';
+
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {intcomma} from 'app/utils';
+import {t} from 'app/locale';
+import {SentryApp} from 'app/types';
+
+type Props = AsyncView['props'];
+
+type State = AsyncView['state'] & {
+  stats: any;
+  errors: any;
+  app: SentryApp | null;
+};
+
+export default class SentryApplicationDashboard extends AsyncView<Props, State> {
+  getDefaultState(): State {
+    return {
+      ...super.getDefaultState(),
+      stats: {},
+      errors: [],
+      app: null,
+    };
+  }
+
+  getEndpoints(): Array<[string, string, any] | [string, string]> {
+    const {appSlug} = this.props.params;
+    if (appSlug) {
+      // Default time range for now: 90 days ago to now
```

Comment:
@nolachen Are the time range parameters required on the backend? If so, we should consider making them optional. I think a case could be made that if the parameters are not passed, it would default to range calculation here (as opposed to querying the full range which could be expensive).

Reply:
`since` and `until` are both optional and provided by the `StatsMixin` endpoint, so the default is 30 days ago to now.

Reply:
@nolachen Then why not just omit the arguments and let the backend handle the defaulting?

Reply:
@scefali Oops the default is actually one day ago to now, so i think the `since` param should just be passed in, especially if in a future version we let the user select the time frame